cfgov/cfgov/urls.py

```diff
@@ -151,19 +151,12 @@ def empty_200_response(request, *args, **kwargs):
         template_name='fair-lending/index.html'),
         name='fair-lending'),
 
-    re_path(
-        r'^consumer-tools/educator-tools/students/knowbeforeyouowe/$',
-        TemplateView.as_view(
-            template_name='students/knowbeforeyouowe/index.html'),
-        name='students-knowbeforeyouowe'
-    ),
-    re_path(
-        r'^consumer-tools/educator-tools/students/'
-        'helping-borrowers-find-ways-to-stay-afloat/$',
-        TemplateView.as_view(
-            template_name='students/helping-borrowers-find-'
-            'ways-to-stay-afloat/index.html'),
-        name='students-helping-borrowers'
+    # KBYO redirect
+    re_path(r'^consumer-tools/educator-tools/students/knowbeforeyouowe/$',
+        RedirectView.as_view(
+            url='/consumer-tools/educator-tools/students/helping-borrowers-find-ways-to-stay-afloat/$', # noqa: E501
+            permanent=True
+        )
```

Comment:
```suggestion
```

What you've done here is told Django to redirect any requests to the URL path:

/consumer-tools/educator-tools/students/knowbeforeyouowe/

permanently to:

/consumer-tools/educator-tools/students/helping-borrowers-find-ways-to-stay-afloat/

Is that what we want to do? I see in your PR description you wrote

> Adds redirect for /knowbeforeyouowe/ to /helping-borrowers-find-ways-to-stay-afloat/

But don't we want /knowbeforeyouowe/ to continue to redirect to the existing Wagtail page https://www.consumerfinance.gov/know-before-you-owe/, as is currently defined in Apache?

Reply:
> Is that what we want to do? 

I think so. This is the existing page https://www.consumerfinance.gov/consumer-tools/educator-tools/students/knowbeforeyouowe/, but in Wagtail the slug is `/consumer-tools/educator-tools/students/helping-borrowers-find-ways-to-stay-afloat/`. But let me know if I'm doing this wrong.

> But don't we want /knowbeforeyouowe/ to continue to redirect to the existing Wagtail page https://www.consumerfinance.gov/know-before-you-owe/, as is currently defined in Apache?

Sorry for the confusion, that PR comment assumes the base path `https://www.consumerfinance.gov/consumer-tools/educator-tools/students/`